lib/order.rb

```diff
@@ -0,0 +1,38 @@
+require_relative 'dish'
+class Order
+  attr_reader :items, :varified
+  def initialize(menu)
+    @items = []
+    @menu = menu
+    @varified = false
+  end
+
+  def varify
+    @varified = true
+  end
+
+  def create_dish(menu_id, dish = Dish.new)
```

Comment:
I find this a bit confusing and I think it's probably a naming issue.  `create_dish` sounds like something that should be used in `Takeaway` or `Menu` but in the context of an `Order` perhaps `add_dish` or `select_dish` would make more sense (to me!).

If this is the only place `Dish` is used, perhaps a better name would be `Item` because orders are 'itemised'.  This is probably more contentious. Creating Dishes in the Order feels wrong though, somehow.

Reply:
I was torn between Item and Dish and landed on Dish - I take your point though this is confusing and is something that I could think about making clearer.